devdocs/settingUp.md

```diff
@@ -72,15 +71,15 @@ Important: When a version is specified, please install that version instead of t
    This command can be run again whenever the dependencies need to be updated.
    Sometimes, the changes from this command might not show up in Eclipse immediately. "Refreshing" the project or restarting Eclipse
    should fix that.
-4. Start Eclipse and go to `File → Import...` and select `Existing Projects into Workspace` under `General`. Set the `root directory` to the location where
+5. Start Eclipse and go to `File → Import...` and select `Existing Projects into Workspace` under `General`. Set the `root directory` to the location where
    the repo is cloned. Click `Finish`.
-5. Start the dev server.<br>
+6. Start the dev server.<br>
```

Comment:
BTW, I think markdown output shows the correct numbers even if the source doesn't have the right numbers. For example, we can use 1 everywhere so that we don't have to update the entire list every time we add/remove an item in the list.